shuffle/prisma/migrations/20230820145837_npx_prisma_generate/migration.sql

```diff
@@ -0,0 +1,5 @@
+-- AlterTable
+ALTER TABLE "FlaggedStatement" ADD COLUMN     "description" TEXT;
+
+-- RenameForeignKey
+ALTER TABLE "Statement" RENAME CONSTRAINT "comments_edited_from_id_fkey" TO "Statement_edited_from_id_fkey";
```

Comment:
Hmm....please rename this migration to make it more meaningful. And we don't need the second part of it anymore.